lib/tuples.gd

```diff
@@ -31,7 +31,7 @@ DeclareInfoClass( "InfoDirectProductElements" );
 ##  <Description>
 ##  <Ref Func="IsDirectProductElement"/> is a subcategory of the meet of
 ##  <Ref Func="IsDenseList"/>,
-##  <Ref Func="IsSmallList"/>,
```

Comment:
Alternatively, perhaps just remove that line, i.e. don't even mention the undocumented name? Since it is not documented, it is not helpful. And technically speaking, the statement is still true w/o mentioning it.